tutor/templates/build/openedx/gunicorn_conf.py

```diff
@@ -0,0 +1,5 @@
+import multiprocessing
+
+# The recommended number of workers is twice the
+# CPU count plus one
+workers = (multiprocessing.cpu_count()-1) * 2 + 2
```

Comment:
Shouldn't we write instead:

    workers = multiprocessing.cpu_count() * 2 + 1

?

Reply:
Oh, yes, we should!

/me hides in shame ;-)